mne/viz/raw.py

```diff
@@ -194,6 +194,14 @@ def plot_raw(raw, events=None, duration=10.0, start=0.0, n_channels=20,
         modes are overrided with ``order`` keyword.
     butterfly : bool
         Whether to start in butterfly mode. Defaults to False.
+    decim : int | 'auto'
+        Amount to decimate the data during display for speed purposes.
+        You should only decimate if the data are sufficiently low-passed,
+        otherwise aliasing can occur. The 'auto' mode (default) uses
+        the decimation that results in a Nyquist frequency at least 1.5 times
+        larger than ``min(info['lowpass'], lowpass)`` (e.g., a 40 Hz lowpass
+        will result in at least a 60 Hz Nyquist frequency and thus 120 Hz
+        displayed sample rate).
```

Comment:
@Eric89GXL can you simplify this sentence as we discussed. Basically a lowpass of 40Hz results results to a decimation up to a sampling rate of 120Hz.